controllers/nnf_clientmount_controller.go

```diff
@@ -197,13 +197,23 @@ func (r *NnfClientMountReconciler) changeMount(ctx context.Context, clientMountI
 
 	if os.Getenv("ENVIRONMENT") == "kind" {
 		if shouldMount {
+			// Return if the directory already exists
+			if _, err := os.Stat(clientMountInfo.MountPath); !os.IsNotExist(err) {
```

Comment:
Couldn't you use `os.IsExist()` here instead?

Reply:
No, that checks for EEXIST which os.Stat() won't return. That made me look at this line again, though, and it's wrong for other reasons. Will fix